### PR TITLE
Fix example for adding local_filter_middleware to middlewares

### PR DESCRIPTION
**Pull request by dylanjw**
### What was wrong?

Related to Issue #1244

### How was it fixed?

Removed the parentheses. 

#### Cute Animal Picture

image

### CODE REVIEW

File: docs/middleware.rst [hunk 1]
```diff
@@ -383,7 +383,7 @@ retrieved using JSON-RPC endpoints that don't rely on server state.
     >>> from web3 import Web3, EthereumTesterProvider
     >>> w3 = Web3(EthereumTesterProvider)
```
Comment (kclowes) [inline docs/middleware.rst]: I think EthereumTesterProvider should have parens too? 

`w3 = Web3(EthereumTesterProvider())`

Comment (dylanjw) [reply docs/middleware.rst]: Doh! Thanks.

File: docs/middleware.rst [hunk 2]
```diff
@@ -381,9 +381,9 @@ retrieved using JSON-RPC endpoints that don't rely on server state.
 .. code-block:: python
```
Comment (pipermerriam) [inline docs/middleware.rst]: Can we update these to be doctests so we catch these types of errors in CI.  Probably good to revert the changes and confirm the doctest failures and then re-apply the fixes to ensure that doctest is doing what it's supposed to.

**Comment by kclowes** [comment]
@dylanjw do you have time to convert this to a doctest? No worries if not :)

**Comment by dylanjw** [comment]
@kclowes I will try to get to it later this week.

**Pull request merged**